main.py

```diff
@@ -468,6 +468,15 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s/batch' % feconf.QUESTION_DATA_URL,
         question.QuestionsBatchHandler),
+    get_redirect_route(
+        r'%s' % feconf.QUESTION_DATA_URL,
+        question.QuestionsHandler),
+    get_redirect_route(
+        r'%s/<collection_id>/<question_id>' % feconf.QUESTION_DATA_URL,
+        question.QuestionsHandler),
```

Comment:
Nope -- one handler per URL please.

Reply:
This is consequence of incapability of delete method to take in payload.
In order to keep uniformity, I will have to change put, post, get methods. Lemme know if you want me to do that.

Reply:
Ah, I see. I'd follow how we do it in editor.py (ExplorationHandler): let's have one URL and handler for GET/PUT/DELETE that has collection_id and question_id as parameters. Then let's have a separately-named URL and handler for POST. Would that work?

Reply:
Yeah, that works.

Reply:
One issue with that, PUT takes in parameters question_id, collection_id, commit message and change_list whereas DELETE takes in collection_id and question_id. If we were to have a common handler for PUT and DELETE while deriving parameters from URL then for the PUT we need to pass the parameters commit message and change_list through payload, which doesn't seem right to me. What do you think?
Also, There is not GET for QuestionHandler.

Reply:
Passing commit message and change list through payload is fine. We do a similar thing in the ExplorationHandler. It keeps the URLs short. Again, I suggest taking a look at ExplorationHandler to see how it's done there, and mimicking it.

Re GET, ah ok, then we don't need that method then.

Reply:
updated.